### ISSUE TITLE
Disable Autobackup

### ISSUE THREAD

**Issue by VieuFou34** [iperov/DeepFaceLab#5643]
Hi everybody !

I'm learning DFL and I don't understand how to disable autobackup during learning process. Even if I set autobackup to every 24H or 0H, the program saves the training. My point is to  be able to cancel steps that are not satisfying, since it's been two weeks and two trainings that are messed up without possibility of going back. After like 400K iterations that were pretty good...
My tests are on a HEAD faceset with SAEHD, with all the possible pretrainings, but the moment I get to work on the colors, all the previous trainings are ruined. 
So is there a way not to automacally save the current training ? And save it like in the morning if I find it ok ?

Thanks very much.

**Comment by JDTCC** [comment]
There is no such way as you describing. One thing you can do however is putting the auto-save at '24H' and just manually saving the model by copying it over to another folder. This way, if you copy it over once a day, you can always go back to previous versions by putting them back, even if they are made months ago.

**Comment by VieuFou34** [comment]
Yep thanks. I gave up with deep face la... I was unable to get some correct result :'(

**Comment by ghost** [comment]
Please close this issue.
